WISE-Community/WISE
# Added About Us page

To test:
- Load `/about` and make sure it looks okay.
- Make sure the links to `/about` appear and work in the header, mobile menu, footer, and features page.

Closes #1465.
